### ISSUE TITLE
Latest version fails to run

### ISSUE THREAD

**Issue by walterra** [thisconnect/test-karma-rollup-tape#23]
For me the latest version isn't working, here's what's happening:

```
git clone git@github.com:thisconnect/test-karma-rollup-tape.git
cd test-karma-rollup-tape
npm install
npm run build
npm run test
```

Here's the output of the `npm run test command`:

```
> test-karma-rollup-tape@1.0.0 test /Users/walterra/dev/test-karma-rollup-tape
> karma start karma.config.js

options.entry is deprecated, use options.input
24 10 2017 10:57:37.298:ERROR [preprocessor.rollup]: Unexpected key 'sourceMap' found, expected one of: acorn, amd, banner, cache, context, entry, exports, extend, external, file, footer, format, globals, indent, input, interop, intro, legacy, moduleContext, name, noConflict, onwarn, output, outro, paths, plugins, preferConst, pureExternalModules, sourcemap, sourcemapFile, strict, targets, treeshake, watch
 at /Users/walterra/dev/test-karma-rollup-tape/test.js
Error: Unexpected key 'sourceMap' found, expected one of: acorn, amd, banner, cache, context, entry, exports, extend, external, file, footer, format, globals, indent, input, interop, intro, legacy, moduleContext, name, noConflict, onwarn, output, outro, paths, plugins, preferConst, pureExternalModules, sourcemap, sourcemapFile, strict, targets, treeshake, watch
    at validateKeys (/Users/walterra/dev/test-karma-rollup-tape/node_modules/rollup/dist/rollup.js:171:11)
    at checkInputOptions (/Users/walterra/dev/test-karma-rollup-tape/node_modules/rollup/dist/rollup.js:10412:14)
    at rollup (/Users/walterra/dev/test-karma-rollup-tape/node_modules/rollup/dist/rollup.js:10483:3)
    at /Users/walterra/dev/test-karma-rollup-tape/node_modules/karma-rollup-plugin/dist/rollup-plugin-karma.js:18:13
    at nextPreprocessor (/Users/walterra/dev/test-karma-rollup-tape/node_modules/karma/lib/preprocessor.js:36:26)
    at /Users/walterra/dev/test-karma-rollup-tape/node_modules/karma/lib/preprocessor.js:122:9
    at module.exports (/Users/walterra/dev/test-karma-rollup-tape/node_modules/isbinaryfile/index.js:28:12)
    at /Users/walterra/dev/test-karma-rollup-tape/node_modules/karma/lib/preprocessor.js:84:7
    at /Users/walterra/dev/test-karma-rollup-tape/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:446:3)
Chrome 61.0.3163 (Mac OS X 10.13.0) ERROR
  You need to include some adapter that implements __karma__.start method!
Firefox 56.0.0 (Mac OS X 10.13.0) ERROR
  You need to include some adapter that implements __karma__.start method!


  ✖ No tests found

...
```

Once I change `sourceMap` to `sourcemap` in `karma.config.js` I still get an error:

```
> test-karma-rollup-tape@1.0.0 test /Users/walterra/dev/test-karma-rollup-tape
> karma start karma.config.js

options.entry is deprecated, use options.input
24 10 2017 10:59:36.431:ERROR [preprocessor.rollup]: bundle.generate(...) now returns a Promise instead of a { code, map } object
 at /Users/walterra/dev/test-karma-rollup-tape/test.js
Error: bundle.generate(...) now returns a Promise instead of a { code, map } object
    at Promise.get (/Users/walterra/dev/test-karma-rollup-tape/node_modules/rollup/dist/rollup.js:10471:9)
    at /Users/walterra/dev/test-karma-rollup-tape/node_modules/karma-rollup-plugin/dist/rollup-plugin-karma.js:20:31
    at process._tickCallback (internal/process/next_tick.js:109:7)
Chrome 61.0.3163 (Mac OS X 10.13.0) ERROR
  You need to include some adapter that implements __karma__.start method!
Firefox 56.0.0 (Mac OS X 10.13.0) ERROR
  You need to include some adapter that implements __karma__.start method!


  ✖ No tests found

...
```

Any ideas how to resolve this?

Thanks,
Walter

**Comment by thisconnect** [comment]
hi, got the same rightnow

I believe it is this regression
https://github.com/TrySound/karma-rollup-plugin/issues/35
https://github.com/TrySound/karma-rollup-plugin/issues/41
actually this plugin is depricated

we should use https://github.com/jlmakes/karma-rollup-preprocessor
Sorry I don't have time, want to try to change to the new one?

**Comment by walterra** [comment]
Thanks for the hints, I was able to fix it and created PR #25.
